docs/getting-started.rst

```diff
@@ -27,7 +27,8 @@ First you need to get web3.js into your project. This can be done using the foll
 - pure js: link the ``dist/web3.min.js``
 
 After that you need to create a web3 instance and set a provider.
-Ethereum supported Browsers like Mist or MetaMask will have a ``ethereumProvider`` or ``web3.currentProvider`` available. For  web3.js, check ``Web3.givenProvider``.
+Ethereum supported Browsers like Mist or MetaMask will have a ``ethereumProvider`` or ``ethereum``
```

Comment:
We can probably remove the reference to `Mist` here since it is deprecated. Can add e.g. "Trust Wallet" which injects a provider.

Reply:
Corrected in a1750bd

Reply:
thx!